File: mpp-firma-transport-c#/FirmaNetworking/DTO/FilterDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FirmaNetworking.DTO
{
    [Serializable]
    public class FilterDTO
    {
        public string destinatie { get; set; }
        public string data { get; set; }

        public FilterDTO(string destinatie, string data)
        {
            this.destinatie = destinatie;
            this.data = data;
        }
    }
}

File: MPP-firma-transport-java/FirmaNetworking/src/main/java/org/example/DTO/FilterDTO.java
package org.example.DTO;
import java.io.Serializable;
import java.time.LocalDateTime;

public class FilterDTO implements Serializable {
    private String destinatie;
    private String data;

    public FilterDTO(String destinatie, String data) {
        this.destinatie = destinatie;
        this.data = data;
    }

    public String getDestinatie() {
        return destinatie;
    }

    public String getData() {
        return data;
    }

    public void setDestinatie(String destinatie) {
        this.destinatie = destinatie;
    }

    public void setData(String data) {
        this.data = data;
    }

    @Override
    public String toString() {
        return "FilterDTO{" +
                "destinatie='" + destinatie + '\'' +
                ", data='" + data + '\'' +
                '}';
    }
}
